security/aws/loadbalancer_internet_facing/README.md

```diff
@@ -1,59 +1,68 @@
-## Check for internet-facing ELBs & ALBs 
- 
-### What it does
+# Check for internet-facing ELBs & ALBs
```

Comment:
Change to **AWS Internet-facing ELBs & ALBs**

Reply:
Changed the header name to AWS Internet-facing ELBs & ALBs